src/main/java/com/ait/lienzo/shared/core/types/Color.java

```diff
@@ -423,23 +412,28 @@ private static final double hueOrPct(String s)
 
     /**
      * Generates a random hex color, e.g. "#1234EF"
-     * 
+     *
      * @return String
      */
     public static final String getRandomHexColor()
     {
-        int r = fixRGB((int) Math.round(Math.random() * 255));
+        int r = randomRGB();
 
-        int g = fixRGB((int) Math.round(Math.random() * 255));
+        int g = randomRGB();
 
-        int b = fixRGB((int) Math.round(Math.random() * 255));
+        int b = randomRGB();
 
         return rgbToBrowserHexColor(r, g, b);
     }
 
+    private static final int randomRGB()
+    {
+        return (int) Math.round(Math.random() * 255);
+    }
```

Comment:
fixRGB() method call was removed because at this case value never will be less then 0 and bigger then 255.